src/lib.rs

```diff
@@ -172,9 +181,9 @@ pub fn build(config: &Config, artifacts: &[&str]) -> Result<()> {
             .stderr(Stdio::piped())
             .spawn()
             .map_err(|e| if e.kind() == io::ErrorKind::NotFound {
-                Error::with_chain(e, format!("frontend `{}` not found", frontend))
+                format_err!("frontend `{}` not found", frontend).into()
```

Comment:
It looks like you could use `chain()` to keep the context here?

Reply:
wait, what's chain()? I don't see it in failure's docs

Reply:
Looks like it's been renamed to `context`? It works slightly differently that `chain_err` did in `error_chain`.

Reply:
Yea I renamed it to context. You could do `e.context(format!(...)).into()` here.